### PR TITLE
fix(components): Print logs for AWS SageMaker components

### PR DESCRIPTION
**Pull request by akartsky**
During code refactoring we missed adding cloudwatch logs functionality to Batch Transform and Processing jobs.
Adding it back now

### CODE REVIEW

**Comment by RedbackThomson** [comment]
lgtm
/ok-to-test

**Comment by surajkota** [comment]
I see only processing and transform, do we have logs for others already checked in ?

**Comment by surajkota** [comment]
Also, is there any quick change we can make to integ tests so this is not missed next time

**Comment by akartsky** [comment]
> I see only processing and transform, do we have logs for others already checked in ?

we only have logging for training, processing and transform jobs

**Comment by akartsky** [comment]
> Also, is there any quick change we can make to integ tests so this is not missed next time

added unit test to make sure `_print_cloudwatch_logs` is called 
and we [already have unit tests](https://github.com/kubeflow/pipelines/blob/master/components/aws/sagemaker/tests/unit_tests/tests/common/test_sagemaker_component.py#L268) for `_print_cloudwatch_logs`

**Comment by surajkota** [comment]
/approve

wait for release changes

File: components/aws/sagemaker/Changelog.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ The version of the AWS SageMaker Components is determined by the docker image ta
 Repository:  https://hub.docker.com/repository/docker/amazon/aws-sagemaker-kfp-components
 
 ---------------------------------------------
+**Change log for version 1.1.0**
```
Comment (surajkota) [inline components/aws/sagemaker/Changelog.md:7]: 1.1.1

**Review by surajkota — CHANGES_REQUESTED** [review]

**Review by surajkota — APPROVED** [review]
/lgtm

**Pull request merged**